mautic/mautic
# Fix progressive forms

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... https://github.com/mautic/mautic/issues/11073



#### Description:
After merge https://github.com/mautic/mautic/pull/8794 we lost some progressive profiling features. This PR fixed it 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a form with 8 fields, set the "Maximum fields displayed at a time" to 4, and make sure to set "Always display" to "No" on all the fields. It should show only 4 fields but it shows all 8.
3. You need these settings for each field
![image](https://user-images.githubusercontent.com/462477/165576537-b7478f37-fe8a-4c5e-8fe9-b0730f20e93d.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11125"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
